doc/classes/TileMap.xml

```diff
@@ -121,9 +121,10 @@
 			<param index="1" name="coords" type="Vector2i" />
 			<param index="2" name="use_proxies" type="bool" default="false" />
 			<description>
-				Returns the [TileData] object associated with the given cell, or [code]null[/code] if the cell does not exist or is not a [TileSetAtlasSource].
+				Returns the [TileData] object associated with the given cell in the related [TileSet], or [code]null[/code] if the cell does not exist or is not a [TileSetAtlasSource].
```

Comment:
Here's my attempt at clarifying this:

Returns the [TileSet]'s [TileData] object for the tile associated with the given cell, or [code]null[/code] if the cell does not exist or is not a [TileSetAtlasSource].

Reply:
Thank you for teaching me that. 🙂